### PR TITLE
refactor(examples) ThankYou page

### PR DESCRIPTION
**Pull request by aniamusial**
# Scope of work
- adjusted styles
- to avoid using calc I sticked to the closest values

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [ ] Self code-reviewed

### CODE REVIEW

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 1]
```diff
@@ -91,42 +125,78 @@ export default {
...
+  flex-direction: column;
   margin: 0 auto;
-  padding: 0 var(--spacer-medium) 0 var(--spacer-medium);
+
+  @include for-desktop {
+    flex-direction: row;
+  }
+
+  &__section {
+    background: var(--c-light);
```
Comment (martaradziszewska) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]: have you considered giving this section full width on mobile as in design? :)

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 2]
```diff
@@ -2,70 +2,104 @@
...
-        transactions, edit subscription to newsletter.
-      </p>
-      <SfHeading title="What can we improve" class="heading sf-heading--left" />
-      <p class="paragraph">
-        Your feedback is important to us. Let us know what we could improve.
-      </p>
-      <div class="wrapper__buttons">
-        <SfButton class="feedback-button color-secondary sf-button--full-width"
-          >SEND MY FEEDBACK</SfButton
+        <SfDivider class="first-divider" />
```
Comment (martaradziszewska) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]: I think divider should be visible on desktop only, see designs on mobile :)

Comment (aniamusial) [reply packages/vue/src/examples/pages/thank-you/ThankYou.vue]: There was a divider before, apparently there was a change in designs ;)

Comment (aniamusial) [reply packages/vue/src/examples/pages/thank-you/ThankYou.vue]: Removing it :)

Comment (pspaczek) [reply packages/vue/src/examples/pages/thank-you/ThankYou.vue]: ... I have one idea, maybe you can delate SfDivider and use border-width: 2px 0. Thanks for that you can remove
```
.first-divider {
  --divider-margin: var(--spacer-base) 0 var(--spacer-sm) 0;
  --divider-border-color: var(--c-white);
}

.second-divider {
  --divider-margin: var(--spacer-sm) 0 0 0;
  --divider-border-color: var(--c-white);
}
```

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 3]
```diff
@@ -135,26 +205,47 @@ export default {
...
+
+.first-divider {
+  --divider-margin: var(--spacer-base) 0 var(--spacer-sm) 0;
+  --divider-border-color: var(--c-white);
+}
+
+.second-divider {
+  --divider-margin: var(--spacer-sm) 0 0 0;
+  --divider-border-color: var(--c-white);
+}
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]:
```suggestion
```
use CSS and remove SfDivider :)

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 4]
```diff
@@ -135,26 +205,47 @@ export default {
...
   }
 }
 .paragraph {
-  font-size: var(--font-size-big);
-  line-height: 1.875rem;
+  font-family: var(--font-family-primary);
+  font-weight: var(--font-light);
+  color: var(--c-dark-variant);
+  margin: 0;
+  line-height: 1.75rem;
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]:
```suggestion
  line-height: 1.6rem;
```
the best l-h for paragraphs 👍

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 5]
```diff
@@ -135,26 +205,47 @@ export default {
...
+    max-width: 100%;
   }
 }
 .paragraph {
-  font-size: var(--font-size-big);
-  line-height: 1.875rem;
+  font-family: var(--font-family-primary);
+  font-weight: var(--font-light);
+  color: var(--c-dark-variant);
+  margin: 0;
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]: p should have margin, maybe 24px top & bottom is k?

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 6]
```diff
@@ -135,26 +205,47 @@ export default {
...
+  &--full {
+    --heading-padding: var(--spacer-base) 0;
+    max-width: 100%;
   }
-
   @include for-desktop {
-    --heading-text-align: left;
+    max-width: 100%;
   }
 }
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]: are we need this CSS? heading always have 100% on this page

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 7]
```diff
@@ -135,26 +205,47 @@ export default {
...
 .feedback-button {
-  margin: var(--spacer) 0;
+  margin: var(--spacer-sm) 0;
 
   @include for-desktop {
-    margin: 0 var(--spacer-medium) 0 0;
+    --button-width: 25rem;
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]: if all buttons on this page have 25rem you can use one place to set it width ;) wdyt

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 8]
```diff
@@ -91,42 +125,78 @@ export default {
...
+    font-family: var(--font-family-primary);
+    font-size: var(--font-sm);
+    font-weight: var(--font-light);
+    @include for-desktop {
+      font-size: var(--font-normal);
+      flex-direction: row;
+    }
   }
 }
 .wrapper {
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]: all elements in this wrapper can be a block, don't wrap it

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 9]
```diff
@@ -2,70 +2,104 @@
...
 export default {
   name: "Product",
   components: {
     SfHeading,
-    SfButton
+    SfButton,
+    SfDivider
   },
   data() {
     return {
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/thank-you/ThankYou.vue]:
```
order: {
number: #80932031-030-00
}
```

File: packages/vue/src/examples/pages/thank-you/ThankYou.vue [hunk 10]
```diff
@@ -2,70 +2,104 @@
   <div id="thank-you">
     <div class="banner">
       <div class="banner__info">
-        <SfHeading title="It's ordered!" class="sf-heading--no-underline" />
-        <p class="banner__order-number">
-          Order No. <strong>#80932031-030-00</strong>
-        </p>
+        <SfHeading
```
[the remark this answers is not in the document]
Comment (pspaczek) [reply packages/vue/src/examples/pages/thank-you/ThankYou.vue]: @aniamusial if you decide to use SfCallToAction here you can add Issues for required improvements 👍

[the remark this answers is not in the document]
Comment (aniamusial) [reply packages/vue/src/examples/pages/thank-you/ThankYou.vue]: Moved to #967 issue

**Comment by pspaczek** [comment]
@aniamusial I added a few small adjustments, it looks ok now. But please:
1. don't duplicate DOM elements, just use CSS to adjust it to design
2. pay more attention to the class name and BEM methodology

**Pull request merged**